pkg/occlient/occlient.go

```diff
@@ -982,6 +982,39 @@ func deleteEnvVars(existingEnvs []corev1.EnvVar, envTobeDeleted string) []corev1
 	return retVal
 }
 
+// ApplyComponentConfig applies the component config onto component dc
+// Parameters:
+//	appName: Name of application of which the component is a part
+//	componentName: Name of the component which is being patched with config
+//	componentConfig: Component configuration
+// Returns:
+//	err: Errors if any else nil
+func (c *Client) ApplyComponentConfig(appName string, componentName string, componentConfig config.ComponentSettings) (err error) {
+	// Get DC name from component name and application name
+	namespacedOpenShiftDC, err := util.NamespaceOpenShiftObject(componentName, appName)
+	if err != nil {
+		return errors.Wrapf(err, "unable to create namespaced name")
+	}
+
+	// Fetch DC matching the component details
+	dc, err := c.GetDeploymentConfigFromName(namespacedOpenShiftDC)
+	if err != nil {
+		return errors.Wrapf(err, "failed to get dc for component %s in application %s", componentName, appName)
+	}
+	glog.V(4).Infof("occlient.go#ApplyComponentConfig got dc \n%+v\n", *dc)
```

Comment:
this should probably be on level 6
see https://kubernetes.io/docs/reference/kubectl/cheatsheet/#kubectl-output-verbosity-and-debugging

Reply:
Ack